samples/net/google_iot_mqtt/src/private_info/create_keys.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
Not sure if this is really needed if we generate .inc file automatically. The openssl commands could be described in readme file instead.

Reply:
we need this for people to generate their own private keys that they can then load to the google iot cloud, or where ever their broker is.